sources/packages/web/src/views/aest/institution/ViewOfferingChangeRequest.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <full-page-container layout-template="centred-tab">
+    <template #header>
+      <header-navigator
+        title="Study period offerings"
+        :routeLocation="{ name: AESTRoutesConst.OFFERING_CHANGE_REQUESTS }"
+        subTitle="View Request"
+      />
+      <program-offering-detail-header
+        class="m-4"
+        :headerDetails="headerDetails"
+      />
+    </template>
+    <template #alerts>
+      <banner class="mb-2" :type="BannerTypes.Warning" :header="bannerText" />
+    </template>
+    <template #tab-header>
+      <v-tabs v-model="tab" color="primary">
+        <v-tab value="requested-change" :ripple="false">Requested Change</v-tab>
```

Comment:
why `:ripple="false"`?

Reply:
To avoid the ripple effect on v-btn when we click between tabs. 
![image](https://user-images.githubusercontent.com/54600590/181345986-4206ed6f-f265-4644-a6e6-79135538213a.png)